sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/TraceHelper.cs

```diff
@@ -171,6 +173,30 @@ internal static string GetOperationName(Activity activity, ref AzMonList MappedT
             return activity.DisplayName;
         }
 
+        internal static string GetV2OperationName(Activity activity, string? url,  ref AzMonList MappedTags)
```

Comment:
nit:  
We will allocate twice for deriving this same value.
1) While setting operation name on telemetry item
2) While setting name on request telemetry

Could be reduce to 1. (Not a blocker, Can be done outside this PR)

Reply:
Yes, I plan on doing this in the third PR where I cover TelemetryItem and its test.